.gitignore

```diff
@@ -11,4 +11,5 @@ env/
 develop-eggs/
 dist/
 *.ipynb_checkpoints/
-
+examples/*.nbconvert.ipynb
```

Comment:
This should be unnecessary.
```
jupyter nbconvert --to notebook --execute foo.ipynb --output foo.ipynb
```
will execute foo.ipynb top to bottom, and overwrite the existing file. It will show you any errors in the output.
```
jupyter nbconvert --clear-output foo.ipynb
```
will remove all output and overwrite the existing file (without executing any code).

Reply:
Fixed, and added these commands to the README